src/RiskModule.sol

```diff
@@ -0,0 +1,111 @@
+/**
+ * Created by Arcadia Finance
+ * https://www.arcadia.finance
+ *
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+pragma solidity >=0.4.22 <0.9.0;
+
+import "../lib/openzeppelin-contracts/contracts/access/Ownable.sol";
+import "./utils/FixedPointMathLib.sol";
+
+/**
+ * @title Risk Module
+ * @author Arcadia Finance
+ * @notice The Risk Module manages the supported asset related risks, collateral factor, liquidity threshold
+ * @dev No end-user should directly interact with the Risk Module
+ */
+contract RiskModule is Ownable {
+    using FixedPointMathLib for uint256;
+
+    mapping(address => mapping(uint256 => uint16)) public collateralFactors;
+    mapping(address => mapping(uint256 => uint16)) public liquidationThresholds;
+
+    uint16 VARIABLE_DECIMAL = 100;
+
+    uint16 MIN_COLLATERAL_FACTOR = 1;
+    uint16 MIN_LIQUIDATION_THRESHOLD = 1;
+
+    uint16 MAX_COLLATERAL_FACTOR = 10000;
+    uint16 MAX_LIQUIDATION_THRESHOLD = 10000;
+
+    uint16 DEFAULT_COLLATERAL_FACTOR = 150;
+    uint16 DEFAULT_LIQUIDATION_THRESHOLD = 110;
+
+    function getCollateralFactor(address assetAddress, uint256 baseCurrency) public view returns (uint16) {
+        require(collateralFactors[assetAddress][baseCurrency] > 0, "RM_GCF: Collateral Factor has to bigger than zero");
```

Comment:
We should not do this check with every get-operation (this will be one of the most used functions), + the check is already covered on the set-operation?

Reply:
How we can be safe, for the scenario that the collateralFactor is not set for asset-base pair?

Reply:
When there is no asset-pair for mapping it returns 0